src/atomgraph.jl

```diff
@@ -90,7 +90,9 @@ function normalized_laplacian(g::G) where G<:lg.AbstractGraph
     a = adjacency_matrix(g)
     d = vec(sum(a, dims=1))
     inv_sqrt_d = diagm(0=>d.^(-0.5))
-    Float32.(I - inv_sqrt_d * a * inv_sqrt_d)
+    lapl = Float32.(I - inv_sqrt_d * a * inv_sqrt_d)
+    @assert !any(isnan.(lapl)) "NaN values in graph Laplacian! This is most likely due to atomic separations larger than the specified cutoff distance leading to block zeros in the adjacency matrix...try increasing the cutoff distance or inspecting your structure to ensure the file is correct."
```

Comment:
`@assert`s are disabled at some optimisation levels
```suggestion
    !any(isnan, lapl) || throw(ArgumentError("NaN values in graph Laplacian! This is most likely due to atomic separations larger than the specified cutoff distance leading to block zeros in the adjacency matrix...try increasing the cutoff distance or inspecting your structure to ensure the file is correct."))
```